code/modules/modular_computers/computers/item/tablet.dm

```diff
@@ -199,6 +199,23 @@
 		return ..()
 	return FALSE
 
+/obj/item/modular_computer/tablet/integrated/get_ntnet_status(specific_action = 0)
+	//No borg found
+	if(!borgo)
+		return FALSE
+	// no AIs/pAIs
+	var/mob/living/silicon/robot/borgs_ONLY = borgo
+	if(!istype(borgs_ONLY))
```

Comment:
Is the variable named "borgo" not guaranteed to be a borg? Stupid names (including yours)

Reply:
It used to be, it isn't anymore as AIs/pAIs were given this when we moved pda to tablet and they needed a messenger app. var name was me being pissed off about it not being borgs only

Reply:
then var name should be renamed to silicon and this variable renamed to borg, and if you dont care enough then just change this to borg_real or something like that but this name is silly